myproject/leave/templates/leave/approval.html

```diff
@@ -0,0 +1,177 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<title>JavaScript - read JSON from URL</title>
+    <script src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
+<style>
+
+header{
+    position:all;
+    font-family: arial, sans-serif;
+    color: maroon;
+}
+ul{
+    position:all;
+    font-family: arial, sans-serif;
+    color: maroon;
+}
+
+h4{
+    font-family: arial, sans-serif;
+    font-size: 14px
+}
+
+p{
+    font-family: arial, sans-serif;
+    color: maroon;
+}
+menu{
+    
+    font-family: arial, sans-serif;
+    color:maroon;
+}
+
+table {
+    font-family: arial, sans-serif;
+    border-collapse: collapse;
+    width: 85%;
+    height: 100%;
+    color:maroon;
+}
+
+td, th {
+    border: 1px solid maroon;
+    text-align: left;
+    font: 20px 5px;
+    padding: 8px;
+}
+
+tr:nth-child(even) {
+    background-color: tan;
+}
+</style>
+
+<script type="text/javascript">
+    var base_url = 'http://127.0.0.1:8000/leave/'; 
+    function getValue(varname)
+    {
+      var url = window.location.href;
+      var vars = {}
+      var hashes = url.split("?")[1];
+      var hash = hashes.split("&");
+      
+      for (i=0; i<hash.length; i++){
+        params = hash[i].split("=");
+        vars[params[0]] = params[1];
+      }return vars;
+    }
+    var username = (getValue()["username"]);
+
+    $(function(){
+        $.ajax({
+            type: 'GET',
+            url: base_url+'approval/'+username+'/?format=json',
+            success: function(data){
+                $.each(data, function(index, data){
+                    $("#employee_name").val(data.employee_name)
+                    $("#reporter").val(data.reporter)
+                	$('#history_of_leave').append("<tr><td>"+data.from_date+"</td>"+
+                		"<td>"+data.to_date+"</td>"+
```

Comment:
You can directly create the tables in the html files. any need for appending it dynamically?